src/Native/System.Native/pal_networking.cpp

```diff
@@ -28,6 +28,11 @@
 #include <alloca.h>
 #endif
 
+#if !HAVE_IN_PKTINFO
+    typedef in_addr in_pktinfo;
```

Comment:
Please CMIIW: I am under the impression that if the statement in `#if` block is not starting with `#`, it should be considered as a CXX code statement and 4-space indentation should be applied.